metadata-ingestion/source-docs-template.md

```diff
@@ -0,0 +1,125 @@
+# Source Name 
+
+<!-- Set Support Status -->
+![Certified](https://img.shields.io/badge/support%20status-certified-brightgreen)
+![Incubating](https://img.shields.io/badge/support%20status-incubating-blue)
+![Testing](https://img.shields.io/badge/support%20status-testing-lightgrey)
+
+## Integration Details
+
+<!-- Plain-language description of what this integration is meant to do.  -->
+<!-- Include details about where metadata is extracted from (ie. logs, source API, manifest, etc.)   -->
+
+### Concept Mapping
+
+<!-- This should be a manual mapping of concepts from the source to the DataHub Metadata Model -->
+<!-- Authors should provide as much context as possible about how this mapping was generated, including assumptions made, known shortcuts, & any other caveats -->
+
+This ingestion source maps the following Source System Concepts to DataHub Concepts:
+
+| Source Concept | DataHub Concept | Notes |
+| -- | -- | -- |
+| eg. Project | eg. Container | eg. This is automatically extracted from the source |
+| eg. View | eg. Dataset | eg. This is modeled as a Dataset with a View subtype |
+| ... | | | 
+
+### Supported Capabilities
+
+<!-- This should be an auto-generated table of supported DataHub features/functionality -->
+<!-- Each capability should link out to a feature guide -->
+
+| Capability | Status | Notes |
+| --- | :-: | --- |
+| Data Container | ✅ | Enabled by default |
```

Comment:
Consider using 🔴 🟡 🟢 mark each feature support as not there, partial (with notes/workarounds), fully (no limitations)

Reply:
That would be problematic for people who might have partial or full color-blindness if we rely on colors in our docs. Suggest plain text if we need three options.

Reply:
Great point, @anshbansal -- I'm going to hold off on making any additional changes here until we move through the template a couple of times